CgfConverter/Services/ArgsHandler.cs

```diff
@@ -334,7 +354,9 @@ public int ProcessArgs(string[] inputArgs)
         public static void PrintUsage()
         {
             Console.WriteLine();
-            Console.WriteLine("cgf-converter [-usage] | <.cgf file> [-outputfile <output file>] [-objectdir <ObjectDir>] [-obj] [-blend] [-dae] [-smooth] [-throw]");
+            Console.WriteLine("cgf-converter [-usage] | <.cgf file> [-outputfile <output file>] [-obj] [-blend] [-dae] [-tif/-png] [-group] [-smooth] [-loglevel <LogLevel>] [-throw] [-dump] [-objectdir <ObjectDir>]");
+            Console.WriteLine();
+            Console.WriteLine("CryEngine Converter v1.3.0");
```

Comment:
The assembly version should be available from a method so this doesn't need to be manually updated.  I try to keep the assembly version updated in the project file, so just reference that with `Assembly.GetExecutingAssembly().GetName().Version` or something similar.

Reply:
Agreed - done